static/src/javascripts/projects/common/modules/email/email-article.js

```diff
@@ -101,34 +116,26 @@ define([
         addListToPage = function (listConfig) {
             if (listConfig) {
                 var iframe = bonzo.create(template(iframeTemplate, listConfig))[0],
-                    $iframeEl = $(iframe),
-                    $insertEl = $(listConfig.insertSelector);
+                    $iframeEl = $(iframe);
 
                 bean.on(iframe, 'load', function () {
                     email.init(iframe);
                 });
 
-                fastdom.write(function () {
-                    $iframeEl[listConfig.insertMethod || 'appendTo']($insertEl && $insertEl.length > 0 ? $insertEl : $articleBody);
-                });
+                if (listConfig.insertMethod) {
+                    fastdom.write(function () {
+                        listConfig.insertMethod($iframeEl);
+                    });
+                } else {
+                    spaceFiller.fillSpace(getSpacefinderRules(), function (paras) {
+                        $iframeEl.insertBefore(paras[0]);
+                        emailInserted = true;
```

Comment:
Given that this is async, is there a risk that `init()` will be called in the interval before spaceFiller has had a chance to run your write function?